app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,70 @@
+class TasksController < ApplicationController
+
+
+  def index
+    @tasks = Task.all
```

Comment:
Just a suggestion, sorting the fields with `.order` would be appropriate to keep the order from changing when the user marks tasks complete.